tallyhowallet/tallyho.org
# Removed CCPA page. Fixes #134

Removed CCPA page on `tallyho.org`. Closes #134 
Modified privacy policy on `tallyho.org/privacy` Closes #133 
Added privacy policy banner on `tallyho.org` Closes #135 135 

**Test**
* Confirm that `https://tallyho.org/ccpa/` is a 404 page.
* Confirm privacy policy matches [this one](https://docs.google.com/document/d/1QT9WDC1RI9XlvcUFoBpRflCoQI04z8CV6P2RoFb1Dhc/edit)
* Confirm all anchor links in Privacy Policy ToC direct to the correct sections
* Confirm link in homepage banner lands on privacy poicy
